src/cpp/src/lm_encoding.cpp

```diff
@@ -125,6 +125,17 @@ std::pair<EncodedResults, int32_t> get_lm_encoded_results(
                                                 active_sequence_groups.end(),
                                                 get_active_sequence_groups),
                                  active_sequence_groups.end());
+    
+    auto stream_generated_tokens = [&streamer_ptr, &generations]() {
+        if (streamer_ptr && generations.at(0).get()->can_read()) {
+            std::unordered_map<uint64_t, GenerationOutput> token = generations.at(0).get()->back();
+            for (const auto& gen_token : token.begin()->second.generated_ids) {
+                if (!streamer_ptr->put(gen_token)) {
```

Comment:
copy-pasted incorrect code? 
According to documentation
https://github.com/openvinotoolkit/openvino.genai/blob/9bcadf7ffdcfe5b133605847d964759593949fac/src/cpp/include/openvino/genai/streamer_base.hpp#L18-L20
we need to break when `put` returns `true`